script.js

```diff
@@ -1,190 +1,139 @@
+      .attr("height", (height + margin.top + margin.bottom));
+      
+var wrapper = svg.append("g").attr("class", "chordWrapper")
+      .attr("transform", "translate(" + (width / 2 + margin.left) + "," + (height / 2 + margin.top) + ")");;
+      
+var outerRadius = Math.min(width, height) / 2  - (mobileScreen ? 80 : 100),
+  innerRadius = outerRadius * 0.95,
+  pullOutSize = (mobileScreen? 20 : 50),
+  opacityDefault = 0.7, //default opacity of chords
+  opacityLow = 0.02; //hover opacity of those chords not hovered over
+  
+var Names = ["The UK","France","Spain", "The US", "The Middle East","","The UK","France","Spain","The US", "The Middle East", ];
+
+var respondents = 95, //Total number of respondents (i.e. the number that makes up the total group)
+  emptyPerc = 0.4, //What % of the circle should become empty
+  emptyStroke = Math.round(respondents*emptyPerc); 
+var matrix = [
+  [0,0,0,0,0,0,15,15,5,5,5,0], //X
+  [0,0,0,0,0,0,5,5,5,5,5,0], //Y
+  [0,0,0,0,0,0,5,2,2,2,2,0], //Z
+  [0,0,0,0,0,0,0,0,0,0,0,emptyStroke], //Dummy stroke
+  [2,1,1,1,1,0,0,0,0,0,0,0], //C
+  [1,1,1,1,1,0,0,0,0,0,0,0], //B
+  [1,1,1,1,2,0,0,0,0,0,0,0], //A
+  [0,0,0,0,0,emptyStroke,0,0,0,0,0,0] //Dummy stroke
+];
+//Calculate how far the Chord Diagram needs to be rotated clockwise to make the dummy
+//invisible chord center vertically
+var offset = (2 * Math.PI) * (emptyStroke/(respondents + emptyStroke))/4;
+
+//Custom sort function of the chords to keep them in the original order
+function customSort(a,b) {
+  return 1;
+};
+
+//Custom sort function of the chords to keep them in the original order
+var chord = customChordLayout() //d3.layout.chord()//Custom sort function of the chords to keep them in the original order
+  .padding(.02)
+  .sortChords(d3.descending) //which chord should be shown on top when chords cross. Now the biggest chord is at the bottom
+  .matrix(matrix);
+  
+var arc = d3.svg.arc()
+  .innerRadius(innerRadius)
+  .outerRadius(outerRadius)
+  .startAngle(startAngle) //startAngle and endAngle now include the offset in degrees
+  .endAngle(endAngle);
+
+var path = stretchedChord()
+  .radius(innerRadius)
+  .startAngle(startAngle)
+  .endAngle(endAngle)
+  .pullOutSize(pullOutSize);
+
+var g = wrapper.selectAll("g.group")
+  .data(chord.groups)
+  .enter().append("g")
+  .attr("class", "group")
+  .on("mouseover", fade(opacityLow))
+  .on("mouseout", fade(opacityDefault));
+
+g.append("path")
+  .style("stroke", function(d,i) { return (Names[i] === "" ? "none" : "#00A1DE"); })
+  .style("fill", function(d,i) { return (Names[i] === "" ? "none" : "#00A1DE"); })
+  .style("pointer-events", function(d,i) { return (Names[i] === "" ? "none" : "auto"); })
+  .attr("d", arc)
+  .attr("transform", function(d, i) { //Pull the two slices apart
+        d.pullOutSize = pullOutSize * ( d.startAngle + 0.001 > Math.PI ? -1 : 1);
+        return "translate(" + d.pullOutSize + ',' + 0 + ")";
+  });
```

Comment:
I'm not sure what style you're going for here, but I'll note that this pullOut stuff is what's causing the current broken appearance.